exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -364,6 +371,45 @@ def _extract_tags_from_dict(self, tags_dict):
             tags[attribute_key] = value
         return tags
 
+    def _extract_tag_value_string_from_sequence(self, sequence):
+        if not sequence:
+            return None
+
+        if self.max_tag_value_length == 1:
+            return None
+
+        tag_value_elements = []
+        running_string_length = (
+            2  # accounts for array brackets in output string
+        )
+        defined_max_tag_value_length = self.max_tag_value_length > 0
+
+        if isinstance(sequence, (list, tuple, range)):
+            for element in sequence:
+                if isinstance(element, (int, bool, float)):
+                    tag_value_element = str(element)
+                elif isinstance(element, str):
+                    tag_value_element = element
```

Comment:
nit L354 to 357 repeated here, could be refactored. One complication is sequence attributes are allowed to have `None`.

Could also combine the two conditions
```suggestion
                if isinstance(element, (int, bool, float, str)):
                    tag_value_element = str(element)
```

Reply:
Simplified the formatting as recommended but didn't refactor the duplication as it's small.

I also changed the logic to silently skip invalid sequence elements versus completely failing so that we're passing along as much data s possible.